src/services/dependencies.service.js

```diff
@@ -1,42 +1,34 @@
 // @flow
+import { PACKAGE_MANAGER_CMD } from './platform.services';
 import * as childProcess from 'child_process';
 
+const spawnProcess = (cmd: string, cmdArgs: string[], projectPath: string) =>
+  new Promise((resolve, reject) => {
+    const child = childProcess.spawn(cmd, cmdArgs, {
+      cwd: projectPath,
+    });
+    child.on(
+      'exit',
+      code => (code ? reject(child.stderr) : resolve(child.stdout))
+    );
+    // logger(child) // service will be used here later
```

Comment:
Not sure what this comment means?

Reply:
Sorry, that it's not clear. 
It's a reminder that we can use the logger service from [logger branch](https://github.com/AWolf81/guppy/blob/logger/src/services/logger.service.js) here for debugging.
It will log every terminal output of the child process to console.